docs/howto/change-highlight-colors.md

```diff
@@ -0,0 +1,93 @@
+# Changing the selection and highlight colors
+
+## Differences between GeoMoose 2.X and 3.X
+
+In GeoMoose 2.X there was a layer called `highlight` that was defined in the mapbook,
+occasionally users would remove this layer and services would cease to function.
+To prevent that from happening GeoMoose 3+ will automatically define a layer
+named `results` and `results-hot`. The `results-hot` layer is used when the user
+has selected a subset of results.
+
+## GeoMoose vector styles
+
+All vector styles are styled using the Mapbox GL Styles format. It is Javascript/JSON
+friendly and well documented here. TODO: INSERT LINK.
```

Comment:
TODO

Reply:
Wow, that's a hard link to find.  MapBox GL Styles are well documented, but I gather we must be using some subset of MapBox GL Styles since overall it looks like it more or less is a JSON version of a GeoMoose 2 mapbook.

Reply:
https://docs.mapbox.com/mapbox-gl-js/style-spec/#layers

If I had to guess it would be some combination of 'paint' and 'layout' properties (although I don't think we are splitting them out into separate categories).

Reply:
Fair enough.  The other thing is if updating it breaks the current styles, it would force us into major release which is somewhat undesirable.